skxray/core/fitting/xrf_model.py

```diff
@@ -1025,10 +1025,14 @@ def construct_linear_model(channel_number, params,
     element_area = {}
 
     for elemental_line in elemental_lines:
+        #print(elemental_line)
         e_model = MS.setup_element_model(elemental_line,
                                          default_area=default_area)
+
+        #MS.assemble_models()
         if e_model:
-            p = e_model.make_params()
+            new_model = MS.compton + MS.elastic + e_model
+            p = new_model.make_params()
```

Comment:
Yeah, I think a v0.9.1 will be coming soon to roll back some accidental API breakages.